src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE web-app PUBLIC
```

Comment:
We shouldn't really need this file since all this is already done with annotations.

Reply:
I think I used it to set the servlet name:
```
        <init-param>
            <param-name>javax.ws.rs.Application</param-name>
            <param-value>org.jboss.resteasy.example.ExampleApp</param-value>
        </init-param>
```
but I could do it differently.

Reply:
Gone